VideoLocker/src/main/java/org/edx/mobile/base/BaseSingleFragmentActivity.java

```diff
@@ -131,24 +131,16 @@ public void finishProcess(){
         hideLoadingProgress();
     }
 
-    public void onMessage(@NonNull MessageType messageType, @NonNull String message){
+    public void onMessage(@NonNull MessageType messageType, @NonNull String message) {
         //TODO - -we need to define different UI message view for different message type?
-        switch ( messageType ){
+        switch (messageType) {
             case FLYIN_ERROR:
```

Comment:
I think we can leave these as they are for the moment, though I guess you should change the call to not make the message sticky, and also remove the margin from the top or convert it to padding (not sure why that was implemented?). I also think these message types provide useful categorizations, and we might have different UI for them at a later time, so I agree that we can hold off on deprecating them for now. As the the new offline/upgrade notification UI, that's just in the form of a persistent banner, and I don't think it will change the existing notification messages UI.